src/igvfd/schemas/construct_library.json

```diff
@@ -0,0 +1,378 @@
+            "type": "string",
+            "enum": [
+                "ATAC-seq",
+                "ChIP-seq",
+                "CUT&RUN",
+                "DNase-seq",
+                "saturation mutagenesis"
+            ]
+        },
+        "quantity": {
+            "title": "Quantity",
+            "description": "The number of elements or guides to be cloned into vectors in the library.",
+            "type": "number",
+            "minimum": 0
+        },
+        "targeted_gene": {
+            "title": "Targeted Gene",
+            "description": "The specific gene(s) this construct library was designed to target.",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "comment": "See gene.json for a list of available identifiers.",
+                "type": "string",
+                "linkTo": "Gene"
+            }
+        },
+        "targeted_locus": {
+            "title": "Targeted Locus",
+            "description": "The specific chromosomal region(s) this construct library was designed to target.",
+            "comment": "Each locus is defined using 1-based, closed coordinates for a specific version of the reference genome assembly.",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "title": "Targeted Locus",
+                "type": "object",
+                "required": [
+                    "assembly",
+                    "chromosome",
+                    "start",
+                    "end"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                    "assembly": {
+                        "title": "Mapping assembly",
+                        "description": "The genome assembly to which coordinates relate (e.g., GRCh38).",
+                        "type": "string",
+                        "enum": [
+                            "GRCh38",
+                            "GRCm39"
+                        ]
+                    },
+                    "chromosome": {
+                        "title": "Chromosome",
+                        "description": "The number (or letter) designation for the chromosome, e.g. chr1 or chrX",
+                        "type": "string",
+                        "pattern": "^(chr[0-9A-Za-z]+)$"
+                    },
+                    "start": {
+                        "title": "Start",
+                        "description": "The starting coordinate.",
+                        "type": "integer",
+                        "minimum": 0
+                    },
+                    "end": {
+                        "title": "End",
+                        "description": "The ending coordinate.",
+                        "type": "integer",
+                        "minimum": 0
+                    }
```

Comment:
I think we need to explicitly say it is 1-based

Reply:
I mentioned it in the overall 'targeted_locus' comment, but I've added "1-based" to each of these sub-properties comments' as well